src/settings.c

```diff
@@ -200,7 +205,7 @@ static HRESULT geolocation_callback(void* context, __FIAsyncOperation_1_Windows_
     IFC(coord->lpVtbl->get_Longitude(coord, &longitude));
 
     float coordinates[2] = {(float)latitude, (float)longitude};
-    SendMessageW(context, WM_GEOLOCATION_UPDATED, 0, (LPARAM)&coordinates);
+    SendMessageW(context, WM_GEOLOCATION_UPDATED, 0, (LPARAM)coordinates);
```

Comment:
This is unrelated and just because I'm in this area anyways. Technically it's been a `float (*)[2]` decayed to `float **`. Luckily it worked when it was casted to `float *` in the dialog callback. Now it's a `float[2]` decayed to `float *`.